docs/source/release_notes.rst

```diff
@@ -7,14 +7,15 @@ Release Notes
     * Fixes
     * Changes
         * Avoid calculating mutualinfo for non-unique columns (:pr:`563`)
+        * Preserve underlying index (:pr:`588`)
```

Comment:
Maybe add a little more detail: "Preserve underlying DataFrame index if index column is not specified"